tests/jballerina-unit-test/src/test/resources/test-src/bala/test_projects/test-union-type-with-null-literal/main.bal

```diff
@@ -0,0 +1,15 @@
+import testorg/testType;
+
+function testPositiveAssignment() {
+    testType:TestType a = func1(testType:doThis);
+}
+
+function func1(function (testType:TestType resp) returns testType:TestType callback) returns testType:TestType {
+    testType:TestType a = null;
+    testType:TestType b = callback(a);
+    return b;
+}
+
+function testImportTypeFromSeparateModule() {
+    testType:A a = null;
+}
```

Comment:
In the positive test, shouldn't we do some assertion?

Reply:
I did not any assertions because problem was triggered when importing the type. Since now the type can import and can use without any errors, I though of not to perform assertion to valideate

Reply:
IMO, it is still better to assert the value, since the crash meant we weren't testing the value part either.